.prettierignore

```diff
@@ -18,6 +18,7 @@
 !*.ts
 !*.tsx
 !*.yml
+!*.sql
```

Comment:
It’s best to keep these extensions alphabetically sorted, but I can do this in https://github.com/hashintel/hash/pull/1874. Not worth re-triggering CI in this PR.